contexts/cloud-messaging.provider.tsx

```
@@ -0,0 +1,78 @@
+  useEffect,
+} from "react";
+import messaging from "@react-native-firebase/messaging";
+import { Alert, Platform } from "react-native";
+
+import { PermissionsAndroid } from 'react-native';
+
+if (Platform.OS === 'android') {
+  PermissionsAndroid.request(PermissionsAndroid.PERMISSIONS.POST_NOTIFICATIONS);
+}
+
+interface CloudMessagingContextProps { }
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)